wgengine/netstack/netstack.go

```diff
@@ -508,6 +508,7 @@ func (ns *Impl) injectInbound(p *packet.Parsed, t *tstun.Wrapper) filter.Respons
 		Data: vv,
 	})
 	ns.linkEP.InjectInbound(pn, packetBuf)
+	packetBuf.DecRef()
```

Comment:
optional: would be nice to add tests for this

Reply:
yeah, still thinking about that. I think the integration tests somehow, but that seems slow & unreliable.

I'd like something tighter.

Reply:
PTAL

Reply:
Thanks!